tests/utils.js

```diff
@@ -0,0 +1,22 @@
+export function deleteCkeditorScripts() {
+	// Give CKE4 some time for destroy actions
+	return delay( 1000, () => {
+		const scripts = Array.from( document.querySelectorAll( 'script' ) );
+		const ckeditorScripts = scripts.filter( scriptElement => {
+			return scriptElement.src.indexOf( 'ckeditor.js' ) > -1;
+		} );
+
+		ckeditorScripts.forEach( x => x.parentNode.removeChild( x ) );
+
+		delete window.CKEDITOR;
+	} );
+}
+
+export function delay( time, func = () => {} ) {
```

Comment:
Wouldn't it be more elegant to have something like:

```javascript
await delay( 1000 );
```

The callback would be invoked just after the delay.

Reply:
It will :) However, this will require having test functions be `async`. And if I applied it - there is a problem to run tests at all. Not sure why 🤔 It is possible due to the docs: https://mochajs.org/#using-async-await, but I've got:
![image](https://user-images.githubusercontent.com/6498567/148848677-9e602b11-3c48-45bc-bc2b-b1b5730b1137.png)

Reply:
It seems to be a bug from Babel and we need an [additional plugin](https://github.com/babel/babel/issues/8829#issuecomment-456524916) if we want to use `async`/`await` syntax (which can, however, result in a slightly bigger bundle).